LFS171x/fabric-material/tuna-app/registerUser.js

```diff
@@ -4,6 +4,10 @@
 */
 /*
  * Chaincode Invoke
+
+Code contributed by the Hyperledger Team 
```

Comment:
Hyperledger Team --> Hyperledger Fabric community

In addition, instead of contributed, we should use same phrasing that you used in the README. Something like "This code is based on code written by the Hyperledger Fabric community."

Gerrit vs. Github here too